internal/controller/csiaddons/persistentvolumeclaim_controller.go

```diff
@@ -63,10 +66,12 @@ var (
 	rsCronJobScheduleTimeAnnotation = "reclaimspace." + csiaddonsv1alpha1.GroupVersion.Group + "/schedule"
 	rsCronJobNameAnnotation         = "reclaimspace." + csiaddonsv1alpha1.GroupVersion.Group + "/cronjob"
 	rsCSIAddonsDriverAnnotation     = "reclaimspace." + csiaddonsv1alpha1.GroupVersion.Group + "/drivers"
+	rsCronJobExcludeAnnotation      = "reclaimspace." + csiaddonsv1alpha1.GroupVersion.Group + "/exclude"
 
 	krcJobScheduleTimeAnnotation = "keyrotation." + csiaddonsv1alpha1.GroupVersion.Group + "/schedule"
 	krcJobNameAnnotation         = "keyrotation." + csiaddonsv1alpha1.GroupVersion.Group + "/cronjob"
 	krCSIAddonsDriverAnnotation  = "keyrotation." + csiaddonsv1alpha1.GroupVersion.Group + "/drivers"
+	krcJobExcludeAnnotation      = "keyrotation." + csiaddonsv1alpha1.GroupVersion.Group + "/exclude"
```

Comment:
instead of exclude, can we have a state that represents state i.e. managed/unmanaged. by default the controller sets it to managed state, and if the user/admin wants they can change the state to unmanaged and modify the CR. if they want to revert back the changes, just set the state back to managed?

Reply:
As an annotation or a spec field?

Reply:
For now annotation only.